tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/api/TestSharedBuffers.java

```diff
@@ -335,38 +378,62 @@ public void testThreeTaskGraphsWithSharedBuffers() throws TornadoExecutionPlanEx
         }
     }
 
-    /**
-     * Tests the persistence of buffers across multiple sequential task graphs with context state management.
-     *
-     * <p>This test demonstrates a four-stage processing pipeline where both intermediate result data and
-     * context state are maintained across task graph executions. It showcases:</p>
-     *
-     * <ul>
-     * <li>Data flow between consecutive task graphs using persistent buffers</li>
-     * <li>Context initialization, updating, and finalization across multiple graphs</li>
-     * <li>Proper pipeline execution with sequential dependencies</li>
-     * <li>The required "touch" operation before transferring results back to host</li>
-     * </ul>
-     *
-     * <p><strong>API Requirements:</strong></p>
-     * <ul>
-     * <li>Buffers must be explicitly marked with {@code persistOnDevice()} to remain accessible</li>
-     * <li>Consuming graphs must use {@code consumeFromDevice()} with the source graph's name</li>
```

Comment:
In this case, we don't need the graph name. The point is to consume data directly from the GPU (or accelerator) from the last tasks executed on the same device.